File: web3rpc/rpc-specs/code-samples/python/debug/vMTracing/traceTransaction.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

txHash = "0x07f6057bc93aca52e53cdbfac9b9830f6a9cae2b3f48f0b47e4cb54959143d09"

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.trace_transaction(txHash)

print(debug_response)

File: web3rpc/rpc-specs/code-samples/javascript/debug/vMTracing/traceTransaction.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const txHash =
    "0x344fc43b5b87984d5a50fe2c54e121f94945ba9ff9da20f9de0f1b4914f47055";

  provider.debug
    .traceTransaction(txHash, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
